src/Engine/UnitActionLifecycles/LoadLifecycle.java

```diff
@@ -168,7 +168,6 @@ public void performEvent(MapMaster gameMap)
         gameMap.removeUnit(unitCargo);
         unitCargo.x = -1;
         unitCargo.y = -1;
-        unitCargo.isTurnOver = true;
```

Comment:
Didn't you just add this? Why are you removing it again?

Reply:
You said `haks r badd`.
So I fixed the thing the hack was bandaging.

The reason I added it was because the MoveEvent was failing and not ending the turn of units that Load... which meant air units could immediately Launch the same turn.

Reply:
OK, now I'm confused. I thought you said you had done this to fix the thing that forced you to move adjacent-then load or something, though in retrospect I'm not sure why that would affect it.
It seems perfectly logical to have turn-ending as a part of a load action; I'm not sure why this would be a hack.
What am I missing (on both sides of this change)?

Reply:
At first, there weren't any particular changes to ensure `unitCargo.isTurnOver == true` around the move->load sequence. This allowed boarding and Launching from a carrier on the same turn.
Then I added a call to snip the move event in load actions (and fiddled with the path snip function in a bad way). This fixed the issue by pointing the loading unit to go adjacent to the transport, and all was well. Then you pointed out that you'd the the same behavior by just moving through a friendly, so I reverted the commit and added the above line (which did work).
(Then I re-updated the snipping in a better way so it should be resilient to pre-emptions on invalid spaces)
Then I made sure `MoveEvent` always sets `unitCargo.isTurnOver = true`, so I reverted this line as now unnecessary.

For the record, the `unitCargo.x/y = -1;` are there because `removeUnit()` won't set them that way if it fails to remove the unit. (this can happen if I e.g. Launch from a Carrier into a Cruiser) I tried changing that in `removeUnit()`, and it made some test cases fail for no apparent reason, so I stopped poking it :P